crowbar_framework/config/locales/crowbar/en.yml

```diff
@@ -551,6 +551,7 @@ en:
         prepare: 'Prepare upgrade'
         download: 'Data download'
         start: 'Start upgrade'
+        install: 'Install Crowbar'
```

Comment:
No, this does not make sense. Why should we talk about *Installing Crowbar* after data restore is done?

![crowbar-upgrade-ui-after-restore](https://cloud.githubusercontent.com/assets/910000/12655015/20cd8e48-c5f7-11e5-8100-49079d687db2.png)